src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,72 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
 
+        createNewFile(toFileName);
+        int[] readFromFileResult = readFromFile(fromFileName);
+        writeContent(readFromFileResult, toFileName);
+
+    }
+
+    private int[] readFromFile(String fromFileName) {
+        try (Scanner scanner = new Scanner(new FileReader(fromFileName))) {
+            int buySum = 0;
+            int supplySum = 0;
+
+            while (scanner.hasNextLine()) {
+                String[] lineContent = scanner.nextLine().split(",");
```

Comment:
make this separator constant. The same for indexs and words 'buy', 'supply' etc.